config/routes.rb

```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
+  get "users/new"
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
-    get "/home", to: "static_pages#home"
-    get "/help", to: "static_pages#help"
-    get "/about", to: "static_pages#about"
+    get  "/home",    to: "static_pages#home"
```

Comment:
```suggestion
    get "/home", to: "static_pages#home"
```
sau `,` để 1 space thôi e